mp/src/game/client/hud_crosshair.h

```diff
@@ -5,55 +5,51 @@
 // $NoKeywords: $
 //=============================================================================//
 
-#ifndef HUD_CROSSHAIR_H
-#define HUD_CROSSHAIR_H
-#ifdef _WIN32
 #pragma once
-#endif
 
 #include "hudelement.h"
 #include <vgui_controls/Panel.h>
 
+#define CWeaponBase C_WeaponBase
+class CWeaponBase;
```

Comment:
You can just do `class C_WeaponBase;` here and for the DrawCrosshair method use `C_WeaponBase` as well.